requirements-M1-python310.txt

```
@@ -0,0 +1,119 @@
+aiohttp==3.8.4
+aiosignal==1.3.1
+anyio==3.6.2
+appdirs==1.4.4
+appnope==0.1.3
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+arrow==1.2.3
+arviz==0.14.0
+asttokens==2.2.1
+async-timeout==4.0.2
+attrs==22.2.0
+backcall==0.2.0
+beautifulsoup4==4.11.2
+bleach==6.0.0
+certifi==2023.5.7
+cffi==1.15.1
+cftime==1.6.2
+charset-normalizer==3.0.1
+clikit==0.6.2
+comm==0.1.2
+contourpy==1.0.7
+crashtest==0.3.1
+cycler==0.11.0
+debugpy==1.6.6
+decorator==5.1.1
+defusedxml==0.7.1
+et-xmlfile==1.1.0
+executing==1.2.0
+fastjsonschema==2.16.2
+fonttools==4.38.0
+fqdn==1.5.1
+frozenlist==1.3.3
+httpstan @ ./httpstan-build-whls/httpstan-4.9.1-cp310-cp310-macosx_12_0_arm64.whl
```

```suggestion
./httpstan-build-whls/httpstan-4.9.1-cp310-cp310-macosx_12_0_arm64.whl
```

I tested with:

```
python3.10 -m venv venv-eurov
source venv-eurov/bin/activate
python -m pip install --upgrade pip
python -m pip install -r requirements-M1-python3.10.txt
```

and it errored on this line. Removing the `httpstan @` made it install nice and smoothly.

It's slightly odd that `pip freeze` generates output that `pip install` can't read back. Anyway, no need to overcomplicate it if this fixes it. 🙂